docs/source/documentation/reconstruction/index.rst

```diff
@@ -6,4 +6,5 @@ This section covers reconstruction methods in scilpy.
 .. toctree::
    :maxdepth: 1
 
-   btensor_scripts
\ No newline at end of file
+   btensor_scripts
+   from_raw_data_to_tractogram
```

Comment:
I think this should be in a new section: complete pipelines. It's not just reconst.

Or we split it and say: see tutorial x for the last section.

@arnaudbore ?